helpers/config-generator.js

```
@@ -0,0 +1,44 @@
+var Promise = require('bluebird');
+var fs = require('fs');
+var prompt = require('prompt');
+
+var readFile = Promise.promisify(fs.readFile);
+var writeFile = Promise.promisify(fs.writeFile);
+
+function run() {
+  prompt.start();
+  const promptOptions = [
+    {
+      name: 'environment',
+      default: 'dev'
+    },
+    {
+      name: 'password',
+      required: true,
+      hidden: true
+    }
+  ]
+  prompt.get(promptOptions, function (err, result) {
```

Might be a good idea to do a 

```js
if (err) {
  throw (err);
}
```

Otherwise it looks like this err will be silently swallowed.